src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -2,15 +2,20 @@
 
 public class AuthenticationService {
     /**
-     * Imagine that some user wants to login to your site.
+     * Imagine that some user wants to log in to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
      * @param email - user's email
-     * @param password - user's password
+     * @param inputtedPassword - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
-    public boolean login(String email, String password) {
+    private final UserService userService = new UserService();
+
+    public boolean login(String email, String inputtedPassword) {
+        if (userService.findByEmail(email) != null && inputtedPassword != null) {
+            return inputtedPassword.equals(userService.findByEmail(email).getPassword());
+        }
```

Comment:
In this case, you call userService.findByEmail(email) twice which is insufficient, because you'll have to iterate over all users twice. It's better to extract this expression to a separate variable